inputs-required.tf

```
@@ -1,6 +1,10 @@
 variable "vm_type" {
   type        = string
   description = "The type of the vm scale set, either windows-scale-set or linux-scale-set"
+  validation {
+    condition     = contains(["windows", "linux"], lower(var.vm_type))
+    error_message = "Unknown VM type. Must be either 'windows' or 'linux'"
```

```suggestion
    condition     = contains(["windows-scale-set", "linux-scale-set"], lower(var.vm_type))
    error_message = "Unknown VM type. Must be either 'windows-scale-set' or 'linux-scale-set'"
```

The module expects these to be either `windows-scale-set` or `linux-scale-set`. There's a couple of places where this has just been copy-pasted throughout the module and not updated to reflect the different expected values